torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2560,6 +2560,37 @@ def generator():
 
     return list(generator())
 
+def sample_inputs_max_pool2d(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    kerneli = [[3, 2], 3]
+    stridei = [[2, 2]]
+    Ni = [1, 4, None]
+    Ci = [32]
+    Hi = [8, 16]
+    Wi = [8, 16]
```

Comment:
this breaks old gradcheck https://app.circleci.com/pipelines/github/pytorch/pytorch/382098/workflows/b671f508-8d28-43e9-a31f-36f17dd72e06/jobs/16090317, biggest sample (4x32x16x16) requires 2GB allocation for old full jacobian check (ninputs x noutputs in double precision comes to 2147483648 bytes, give or take).
Samples for OpInfos should be smaller, given the gradchecks that are run.

Reply:
Sending a PR to fix that by reducing all the sizes significantly.

Reply:
sorry about that ! i think i copied these inputs from another test but those tests must not have been doing gradgrad..